packages/varnish/src/components/header/AI2Banner.tsx

```diff
@@ -1,10 +1,14 @@
 import * as React from 'react';
 import styled from 'styled-components';
-import { AI2Logo } from '../logos/AI2Logo';
+import Grid from '@mui/material/Grid';
+import { AI2Logo, LogoSize } from '../logos/AI2Logo';
 import { Content } from '../Content';
 
 export interface AI2BannerProps {
   children?: React.ReactNode;
+  size?: LogoSize;
+  controls?: React.ReactNode;
```

Comment:
consider renaming to endSlot or something... the user can pass whatever... not just controls